client/src/worldgen.rs

```diff
@@ -256,6 +257,17 @@ impl ChunkParams {
         }
     }
 
+    fn generate_road_support(&self, center: na::Vector3<f64>) -> Option<Material> {
+        let plane = -Plane::from(Side::B);
+        let horizontal_distance = plane.elevation(center, self.chunk);
+
+        if horizontal_distance.abs() < 0.3 {
```

Comment:
You don't actually need to call `abs()` here. Neighboring nodes have coordinate systems reflected across the common face, so the nodes on either side of the road plane have distances with the same sign.